lib/iris/tests/test_plot.py

```diff
@@ -131,23 +127,25 @@ def test_orography(self):
         self.check_graphic()
 
         # TODO: Test bounds once they are supported.
-        with self.assertRaises(NotImplementedError):
-            qplt.pcolor(self.cube)
+        qplt.pcolor(self.cube)
+        with pytest.raises(NotImplementedError):
             iplt.orography_at_bounds(self.cube)
-            iplt.outline(self.cube)
-            self.check_graphic()
+            # iplt.outline(self.cube)
```

Comment:
commented code

Reply:
I believe this commented code is here to signify how the test should look once the `# TODO` has been addressed.

https://github.com/SciTools/iris/blob/098466411b26e8c539028569b55686fea1ad607c/lib/iris/tests/test_plot.py#L129

Is there a better format that you would prefer?